src/components/FeedCard/FeedCard.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { useState } from 'react';
+import { ContentsWrapper } from '@src/components/FeedCard/FeedCard.style';
+import Avatar from '@mui/material/Avatar';
+import {
+  Card,
+  CardHeader,
+  IconButton,
+  CardContent,
+  Typography,
+  CardMedia,
+  Box,
+  CardActions,
+} from '@mui/material';
+import FavoriteBorderIcon from '@mui/icons-material/FavoriteBorder';
+import FavoriteIcon from '@mui/icons-material/Favorite';
+import FeedTypographyButton from '@components/FeedCard/FeedTypographyButton/FeedTypographyButton';
+import FeedMenuIconButton from '@components/FeedCard/FeedMenuIconButton/FeedMenuIconButton';
+
+function FeedCard() {
+  const [contentsOpen, setContentsOpen] = useState(false);
+  const [liked, setLiked] = useState(false);
+
+  const handleReadMore = (e: React.MouseEvent<HTMLElement>) => {
+    setContentsOpen(true);
+  };
+
+  const handleLiked = (e: React.MouseEvent<HTMLElement>) => {
+    //좋아요 부분
+    setLiked(!liked);
+  };
+
+  return (
+    <Card sx={{ width: '608px', borderRadius: '8px' }}>
```

Comment:
`Card` 에는 `MuiProvider`에서 적용되는 `borderRadius`가 있기 때문에 해당 내용 제거하셔도 될 것 같습니다!

```suggestion
    <Card sx={{ width: '608px' }}>
```

Reply:
삭제 완료했습니다. :) 감사합니다.